packages/service-worker/worker/test/happy_spec.ts

```diff
@@ -1759,6 +1765,66 @@ describe('Driver', () => {
          });
     });
   });
+
+  describe('freashnessForNavigationUrls', () => {
+    it('doesn\'t create navigate request if when disabled', async () => {
```

Comment:
```suggestion
    it('doesn\'t create navigate request when disabled', async () => {
```

Also, I think the following description would be a little more clear:

```suggestion
    it('serves the request from the cache when disabled', async () => {
```

Reply:
Oh, nice catch with `if when` 😂 I guess I was changing message half way trough.

---

Regarding the suggested message:
> serves the request from the cache when disabled

I think it's not 100% accurate 🤔 By default, the navigation request is not cached nor requested, rather it's replaced by the `/index.html` request here:
```ts
return this.handleFetch(this.adapter.newRequest(this.indexUrl), context);
```

Maybe this is not important and the suggested message is fine. WDYT?